x-pack/plugins/security_solution/public/management/links.ts

```diff
@@ -204,6 +221,19 @@ export const getManagementFilteredLinks = async (
     (canReadHostIsolationExceptions &&
       (await checkArtifactHasData(HostIsolationExceptionsApiClient.getInstance(core.http))));
 
+  if (ExperimentalFeaturesService.get()?.riskScoringRoutesEnabled) {
+    const existingCategories = links.categories ?? [];
+    links.categories = [
```

Comment:
Could you briefly tell me what was the purpose of this? Seems a bit confusing, but it's probably because I havent had a coffee yet today ;p Thanks!

Reply:
It's add the whole EA section (category) for this menu,
As we aim to release at 8.10, we can remove this code before that
<img width="1005" alt="Screenshot 2023-07-03 at 13 22 08" src="https://github.com/elastic/kibana/assets/7609147/76119a80-c7e4-4745-89db-9019f2264092">

Reply:
I see thanks! 
What do you think about adding category directly here: https://github.com/elastic/kibana/blob/aad68003a62d1329755f10221a18db1258a5c4eb/x-pack/plugins/security_solution/public/management/links.ts#L51 
hidden behind a eperimental flag condition? 

I think that would be much more readable.

Reply:
If we move out there can be errors, because we not yet initialise Feature flag service.
I understand that code is not ideal, but intention to remove this FF, and move the category to array before there release

Reply:
I see the `SecurityPageName.entityAnalyticsManagement` LinkItem definition already has the `experimentalKey: 'riskScoringRoutesEnabled'` defined, this properly will automatically filter the link if the experimental key is not enabled. 
There's no need to conditionally add a category, `categories` should be static, if there's no link enabled in the category it will not be displayed.